CP01CNSP/CP01CNSP_R00002_ingest.csv

```diff
@@ -0,0 +1,13 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.dbg_pdbg.cspp.dbg_pdbg_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/*PDBG.txt,CP01CNSP-SP001-00-SPPENG000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.wc_hmr.cspp.wc_hmr_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/*WC_HMR.txt,CP01CNSP-SP001-00-SPPENG000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.wc_sbe.cspp.wc_sbe_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/*WC_SBE.txt,CP01CNSP-SP001-00-SPPENG000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.wc_wm.cspp.wc_wm_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/*WC_WM.txt,CP01CNSP-SP001-00-SPPENG000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.optaa_dj.cspp.optaa_dj_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_ACS_ACS.txt,CP01CNSP-SP001-02-OPTAAJ000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.nutnr_j.cspp.nutnr_j_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_SNA_SNA.txt,CP01CNSP-SP001-03-NUTNRJ000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.velpt_j.cspp.velpt_j_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_PPB_ADCP.txt,CP01CNSP-SP001-05-VELPTJ000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.dosta_abcdjm.cspp.dosta_abcdjm_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_PPB_OPT.txt,CP01CNSP-SP001-06-DOSTAJ000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.spkir_abj.cspp.spkir_abj_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_PPB_OCR.txt,CP01CNSP-SP001-07-SPKIRJ000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.ctdpf_j.cspp.ctdpf_j_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_PPB_CTD.txt,CP01CNSP-SP001-08-CTDPFJ000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.flort_dj.cspp.flort_dj_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_PPB_TRIP.txt,CP01CNSP-SP001-09-FLORTJ000,recovered_cspp,Not Available,failed deployment; no data available
+#mi.dataset.driver.parad_j.cspp.parad_j_cspp_recovered_driver,/omc_data/whoi/OMC/CP01CNSP/R00002/extract/ucspp_*_PPB_PARS.txt,CP01CNSP-SP001-10-PARADJ000,recovered_cspp,Not Available,failed deployment; no data available
```

Comment:
Looks good. I know there is no data, but should some of these refer to "fixed" rather than "extract" like for R0004?